src/style.css

```diff
@@ -0,0 +1,64 @@
+.frame {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars--0 {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars--1 {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars--2 {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars--3 {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars--4 {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars--5 {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-size: cover;
+  box-sizing: border-box;
+}
+
+.active-star {
+  background-image: url("./images/star-active.svg");
+
+}
```

Comment:
You don't need to do the active stars class. You need to do this through the nth-child and the number in the class

Reply:
Can you give me an example, because i don't get how to do this through the nth-child and the number in the class. Ty